src/dotnet/Microsoft.DotNet.Core.Sdk.props

```diff
@@ -15,7 +15,8 @@
 
   <!-- User-facing configuration-specific defaults -->
   <PropertyGroup>
-    <PlatformTarget>AnyCPU</PlatformTarget>
+    <!-- TODO: This property can't be set in the CLI since ResolveNuGetPackageAssets needs an exact RID from the project.json, and the RID is made up of the PlatformTarget -->
+    <!--<PlatformTarget>AnyCPU</PlatformTarget>-->
```

Comment:
For now, the CLI passes this in during "build3".  We need the new "ResolveNuGetPackageAssets" target that Nat is writing in order to make this correct.

---
In reply to: [71824700](https://github.com/dotnet/cli/pull/3911#discussion_r71824700) [](ancestors = 71824700)